set.mm

```diff
@@ -815565,306 +816034,612 @@ two graphs which are not (ordinary) isomorphisms between these graphs, as
-      JUVNUWFUUDUVHEUWFUUBUVFUUCUVGRYTUVDWRZUMYTUVDWRZSTUWFUUGUVKEUWFUUBUVFUUFU
-      VJUWGUNYTUVDWRZSTUWFUUIUVMEUWFUUCUVGUUFUVJUWHUWISTWLXGWSWPYOXNUUNVGZYIYKU
-      WJYNYKXNDYSUANZYTULZUUKQZJUOZUAYQWTZMZUUNYKFVNMZXNUWPVGFXOGHXAUWQXMUWODUA
-      JEFGVNHIXBXCVDUWNUUMUADYQUWKDOZUWMUULJUWRUWLUUAUUKUWKDYSYTWGXDXEXFXHVFVFV
-      JXIXJXKXL $.
-  $}
+    $d G x y z $.
+    $( ` G ` contains a triangle ` 0 , 1 , 2 ` , with corresponding edges
+       ` { 0 , 1 } , { 1 , 2 } , { 0 , 2 } ` .  (Contributed by AV,
+       3-Aug-2025.) $)
+    usgrexmpl1tri $p |- { 0 , 1 , 2 } e. ( GrTriangles ` G ) $=
+      ( cc0 c1 c2 ctp wcel wceq c3 cpr w3a wo orci elun mpbir eleq1d cgrtri cfv
+      vx vy vz cv chash csn c4 cun wrex c0ex tpid1 1ex tpid2 tpid3 3pm3.2i eqid
+      c5 2ex ex-hash prex olci tpeq1 preq1 biidd 3anbi123d 3anbi13d tpeq2 preq2
+      eqeq2d tpeq3 rspc3ev cvtx usgrexmpl1vtx eqcomi cedg usgrexmpl1edg isgrtri
+      mp2an ) GHIJZBUAUBKWAUCUFZUDUFZUEUFZJZLZWAUGUBMLZWBWCNZGMNUHZGHNZGINZHINZ
+      JZMUINMUSNUIUSNJZUJZUJZKZWBWDNZWPKZWCWDNZWPKZOZOZUEWAMUIUSJZUJZUKUDXEUKUC
+      XEUKZGXEKZHXEKZIXEKZOWAWALZWGWJWPKZWKWPKZWLWPKZOZOZXFXGXHXIXGGWAKZGXDKZPX
+      PXQGHIULUMQGWAXDRSXHHWAKZHXDKZPXRXSGHIUNUOQHWAXDRSXIIWAKZIXDKZPXTYAGHIUTU
+      PQIWAXDRSUQXJWGXNWAURVAXKXLXMXKWJWIKZWJWOKZPYCYBYCWJWMKZWJWNKZPYDYEWJWKWL
+      GHVBUMQWJWMWNRSVCWJWIWORSXLWKWIKZWKWOKZPYGYFYGWKWMKZWKWNKZPYHYIWJWKWLGIVB
+      UOQWKWMWNRSVCWKWIWORSXMWLWIKZWLWOKZPYKYJYKWLWMKZWLWNKZPYLYMWJWKWLHIVBUPQW
+      LWMWNRSVCWLWIWORSUQUQXCXOWAGWCWDJZLZWGGWCNZWPKZGWDNZWPKZXAOZOWAGHWDJZLZWG
+      XKYSHWDNZWPKZOZOUCUDUEGHIXEXEXEWBGLZWFYOXBYTWGUUFWEYNWAWBGWCWDVDVKUUFWQYQ
+      WSYSXAXAUUFWHYPWPWBGWCVETUUFWRYRWPWBGWDVETUUFXAVFVGVHWCHLZYOUUBYTUUEWGUUG
+      YNUUAWAWCHGWDVIVKUUGYQXKXAUUDYSUUGYPWJWPWCHGVJTUUGWTUUCWPWCHWDVETVHVHWDIL
+      ZUUBXJUUEXNWGUUHUUAWAWAWDIGHVLVKUUHXKXKYSXLUUDXMUUHXKVFUUHYRWKWPWDIGVJTUU
+      HUUCWLWPWDIHVJTVGVHVMVTUCUDUEWAWPBXEBVNUBXEABCDEFVOVPBVQUBWPABCDEFVRVPVSS
+      $.
+  $}
+
+  $( Lemma 2 for ~ usgrexmpl2nb0 etc.  (Contributed by AV, 9-Aug-2025.) $)
+  usgrexmpl2nblem2 $p |- ( { K , N } e. ( { { 0 , 3 } }
+                                u. ( { { 0 , 1 } , { 1 , 2 } , { 2 , 3 } }
+                                   u. { { 3 , 4 } , { 4 , 5 } , { 0 , 5 } } ) )
+                            <-> ( { K , N } = { 0 , 3 }
+                      \/ ( ( { K , N } = { 0 , 1 } \/ { K , N } = { 1 , 2 }
+                                 \/ { K , N } = { 2 , 3 } )
+                           \/ ( { K , N } = { 3 , 4 } \/ { K , N } = { 4 , 5 }
+                                 \/ { K , N } = { 0 , 5 } ) ) ) ) $=
+    ( cpr cc0 c3 csn c1 c2 ctp c4 c5 cun wcel wceq w3o elun eltp orbi12i bitri
+    wo prex elsn ) ABCZDECZFZDGCZGHCZHECZIZEJCZJKCZDKCZIZLZLMUCUEMZUCUNMZTUCUDN
+    ZUCUFNUCUGNUCUHNOZUCUJNUCUKNUCULNOZTZTUCUEUNPUOUQUPUTUCUDABUAZUBUPUCUIMZUCU
+    MMZTUTUCUIUMPVBURVCUSUCUFUGUHVAQUCUJUKULVAQRSRS $.
```

Comment:
The same theorem could be written in a more general way and added to the family of `elsn`, `elpr`, `eltp`. 
Maybe `el7p` for "element to a 7-uple" ?

Reply:
(Using `A` in place of `{ K , N }`, `B` in place of `{ 0 , 3 }`, `C` in place of `{ 0 , 1 }`, etc.)